src/signatures.jl

```diff
@@ -235,20 +237,22 @@ function identify_framemethod_calls(frame)
     return methodinfos, selfcalls
 end
 
-# try to normalize `def` to `Symbol` representation
-function normalize_defsig(@nospecialize(def), frame::Frame)
+# try to normalize `def` to `GlobalRef` representation
+function normalize_defsig(@nospecialize(def), mod::Module)
     if def isa QuoteNode
         def = nameof(def.value)
-    elseif def isa GlobalRef
-        def = def.name
+    end
+    if def isa Symbol
+        def = GlobalRef(mod, def)
```

Comment:
The `GlobalRef` constructor specifically? I don't see why that would cause any issues, it's well typed. As for the overall method, it still returns `Any` as before since there's a fallthrough path.

Reply:
No, the method as a whole. I forgot this was added in #66, so no worries.